submissions/aymkin/OOP-JS (tjsw)/index.js

```diff
@@ -0,0 +1,80 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+
+Code repository: https://github.com/aymkin/a-tiny-JS-world/tree/tiny-js-world-oop-aymkin
+Web app: https://aymkin.github.io/a-tiny-JS-world/
+   */
+
+function Inhobitant(species, name, gender, friends, say) {
```

Comment:
There are more chances that an inhabitant has something to say than has got any friends. As per requirements every inhabitant says something and not every inhabitant has friends.

By convention mandatory parameters come first, optional follow.

Under current implementation this means that when we do not want an inhabitant to have friends we still need to specify those, i.e.
`let inhabitant = new Inhabitant('giraffe', 'Griggio', 'male', undefined, 'whatever giraffes say');`

Alternative `let inhabitant = new Inhabitant('giraffe', 'Griggio', 'male', 'whatever giraffes say');` where `friends` param comes at the end would look more consistent.

Reply:
Done:
* parameter `friends` in constructors has been moved to the last 
* check is it friends array empty to reflect the right message